analysis/warriorfury/src/modules/shadowlands/legendaries/NaturesFury.tsx

```diff
@@ -0,0 +1,55 @@
+import SPELLS from 'common/SPELLS';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+class NaturesFury extends Analyzer {
+  _ancientAftershockDamage: number = 0;
+  _naturesFuryDamage: number = 0;
+  _hasLegendary: boolean = this.selectedCombatant.hasLegendary(SPELLS.NATURES_FURY);
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasCovenant(COVENANTS.NIGHT_FAE.id) || this._hasLegendary;
```

Comment:
why does this key off of the covenant as well as the legendary? we can't rely on people having Unity (unfortunately)

Reply:
Ahh right. No need to check for covenant and legendary.

Fixed. Separated Ancient Aftershock from the Nature's Fury module.